docs/swagger.yaml

```diff
@@ -458,20 +552,62 @@ components:
         - LITRE
       description: Available units for quantity. Will default to PCS if not set
       default: PCS
+    ErrorResponse:
+      description: ''
+      type: object
+      x-examples:
+        example-1:
+          type: 'https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1'
+          title: One or more validation errors occurred.
+          detail: BadRequest
+          instance: 00-616e415f5af7584eb4d9369d0fe8d7cc-48b90041faff204c-00
+          invalidParams:
+            - name: RedirectUrl
+              reason: 'The field RedirectUrl must match the regular expression ''^https://[\w\.]+([\w#!:.?+=&%@\-/]+)?$''.'
+      properties:
+        type:
+          type: string
+          minLength: 1
+        title:
+          type: string
+          minLength: 1
+        detail:
+          type: string
+          minLength: 1
+        instance:
+          type: string
+          minLength: 1
+        invalidParams:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+                minLength: 1
+              reason:
+                type: string
+                minLength: 1
+            required:
+              - name
+              - reason
+      required:
+        - title
+        - detail
+        - instance
+    OrderV2Response:
```

Comment:
maybe call this "getOrderResponse"?

Reply:
Synes ikke det. Vil gjerne kunne gjenbruke dette dersom man f.eks skulle returnere en lite av orderResponse. Da blir det litt rart med Array [getOrderResponse]
Vil også kalle den V2. Slik at det blir eksplisitt at denne hører til V2